src/script/components/ConfigToolbar/ConfigToolbar.tsx

```diff
@@ -34,15 +35,14 @@ export function ConfigToolbar() {
 
   // Toggle config tool on 'cmd/ctrl + shift + 2'
   useEffect(() => {
-    const handleKeyDown = (event: KeyboardEvent) => {
-      if (isMetaKey(event) && event.shiftKey && event.key === '2') {
```

Comment:
note that `isMetaKey` isn't the problem, `(event.metaKey || event.ctrlKey)` doesn't work either